tabby-electron/src/services/platform.service.ts

```diff
@@ -208,12 +225,29 @@ export class ElectronPlatformService extends PlatformService {
             paths = result.filePaths
         }
 
-        return Promise.all(paths.map(async p => {
-            const transfer = new ElectronFileUpload(p, this.electron)
-            await wrapPromise(this.zone, transfer.open())
-            this.fileTransferStarted.next(transfer)
-            return transfer
-        }))
+        if(options.directory) {
+            let allFiles: string[] = []
```

Comment:
An array of `{ fullPath: string, relativePath: string }` would be cleaner than two separate `string[]`s